README.md

```diff
@@ -170,3 +171,16 @@ While you **can** target the RISC-V Espressif SOCs (`esp32-cXX` and `esp32-hXX`)
 You need a Python 3.7 or later installed on your machine.
 * Linux, Mac OS X: if not preinstalled already, just install it with your package manager
 * Windows: install it e.g. [from the official Python site](https://www.python.org/downloads/).
+
```

Comment:
Do you want to push the "More information" section above "Prerequisites"? Just asking... Otherwise the user might not even see it.

Reply:
yeah maybe that is a better place